adapters/has_component_adapter.py

```diff
@@ -0,0 +1,93 @@
+from adapters.compound_adapter import CompoundAdapter
+import rdflib
+from owlready2 import *
+from tqdm import tqdm
+from adapters import Adapter
+from biocypher._logger import logger
+from adapters import *
+
+
+class HasComponentEdge(CompoundAdapter):
+
+    def __init__(self, filepath=None, label="has_component", dry_run=False):
+        super().__init__(filepath=filepath, label=label, dry_run=False)
+        self.type = "edge"
```

Comment:
What is the purpose of this `type` variable?

Reply:
> What is the purpose of this `type` variable?

nothing. I've omitted it and also the type checking logic too. as we're writing edge adapter there is no need to check if the type is edge or node.